src/components/popup/intro/footer.jsx

```diff
@@ -36,6 +36,14 @@ export default class IntroFooter extends Component {
             store
         } = props;
 
+        let allPurposes = [];
+        if (store && store.vendorList && store.vendorList.purposes) {
```

Comment:
do you really need to check `if (store)` ?

Reply:
Prooooobably not (what are the chances that it wouldn't be there), but it makes me feel better that we have that case handled. Kinda like putting a battery in a solar powered watch.